core-bundle/src/DependencyInjection/ContaoCoreExtension.php

```diff
@@ -255,4 +257,18 @@ private function overwriteImageTargetDir(array $config, ContainerBuilder $contai
 
         @trigger_error('Using the "contao.image.target_path" parameter has been deprecated and will no longer work in Contao 5.0. Use the "contao.image.target_dir" parameter instead.', E_USER_DEPRECATED);
     }
+
+    private function handleTokenCheckerConfig(array $config, ContainerBuilder $container): void
+    {
+        if (!$container->hasDefinition('contao.security.token_checker')) {
+            return;
+        }
+
+        $tokenChecker = $container->getDefinition('contao.security.token_checker');
+        $tokenChecker->replaceArgument(5, new Reference('security.access.simple_role_voter'));
+
+        if ($container->hasParameter('security.role_hierarchy.roles') && 0 < \count($container->getParameter('security.role_hierarchy.roles'))) {
```

Comment:
`0 < \count()` means `\count() > 0`, doesn't it? The yoda condition is confusing me a lot here. 😄

Reply:
Yes. Looks totally stupid, I know 😅.

Reply:
The is covered by a testcase anyway.

Reply:
It is also unnecessary, because the purpose of yoda conditions is to prevent accidental variable assignments such as `if ($a = getB())` instead of `if ($a == getB())`. This can never happen with the lesser/greater than operator, therefore we can use `\count() > 0` here.

Reply:
Sure, but isn't the CS fixer going to revert this again anyway?

Reply:
Edit: it doesn't! 🎉